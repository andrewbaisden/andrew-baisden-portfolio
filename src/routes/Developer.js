import React from 'react';
import { Helmet } from 'react-helmet';
import Header from '../components/Header';
import Footer from '../components/Footer';
import developIcon from '../img/developIcon.svg';
import MenuServicesDev from '../components/MenuServicesDev';
import ServicesDeveloperPod from '../components/ServicesDeveloperPod';
import ServicesBreadcrumb from '../components/ServicesBreadcrumb';
import CheckAboutPage from '../components/CheckAboutPage';

const Developer = () => (
  <div>
    <div>
      <Helmet>
        <meta
          name="description"
          itemProp="description"
          content="Over the years I have constantly being evolving my skills, and I have now settled on a very modern developer workflow. It consists of using the Javascript frameworks React and Vue."
        />
        <title>Developer - About - Full Stack Developer | Andrew Baisden | London</title>
        <link rel="canonical" href="https://andrewbaisden.com/about/developer" />
      </Helmet>
    </div>
    <CheckAboutPage />
    <Header />
    <div className="developer-page-container">
      <ServicesBreadcrumb />
      <div className="developer-page-content">
        <div className="services-home">
          <h1>Services</h1>
          <img src={developIcon} alt="Developer" />
          <p>Developer</p>
        </div>
      </div>

      <MenuServicesDev menuBG="var(--base-colour-orange)" />
      <ServicesDeveloperPod />
    </div>
    <Footer />
  </div>
);

export default Developer;
