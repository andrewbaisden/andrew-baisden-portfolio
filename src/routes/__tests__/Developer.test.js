import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Developer from '../Developer';

Enzyme.configure({ adapter: new Adapter() });

describe('<Developer />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<Developer />);
    expect(wrapper).toMatchSnapshot();
  });
});
