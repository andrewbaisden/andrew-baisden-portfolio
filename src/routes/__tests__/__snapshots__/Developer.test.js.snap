// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Developer /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
